### PR TITLE
SUP-3054 removed refs to pcp, clean up distribute methods docs, added…

### PR DESCRIPTION
**Pull request by taikaa**
SUP-3054 

### Prior to this commit: 
It was not apparent to ca_extend users that they did not need to use ssh/winrm to distribute the CA certificate to agents, if the CA certificate has not yet expired. Additionally, the **Configuration** section did not make clear at which point in the process users make the decision to use an Inventory file, connect Bolt to Puppet DB, or use ~/.ssh/config. The README also made references to PCP transport without details on which tasks/plans are supported to run via PCP.

### This commit:
Adds an additional option for distributing a copy of CA cert to agents, using a Puppet `file` resource along with including an example of puppet code using this option. There is now a total of three suggested methods for distribution, and these methods are now logically located under the **Usage**. What was formerly the **Configuration** section has been logically relocated to the usage instructions for **1. Using the ca_extend::upload_ca_cert Plan** because the references to inventory files, transport configuration, and Puppet DB belongs here. Removes references to PCP transport because not all tasks/plans in `ca_extend` at this time support use via the Orchestrator.

### CODE REVIEW

**Review by m0dular — CHANGES_REQUESTED** [review]
Looks great, just a minor note about the `rm` example.

**Comment by taikaa** [comment]
Which change should be made to the rm example? Remove -r or change something else? Thanks! @m0dular

File: README.md [hunk 1]
```diff
@@ -179,14 +161,50 @@ Note that if you are running `extend_ca_cert` locally on the primary Puppet serv
...
 ```
 
+#### 2. Manually deleting `ca.pem` on agents and letting them download that file as part of the next Puppet agent run
+
+The agent will download `ca.pem` only if it is absent, so it must be deleted to use this method. 
+
+For example, on an \*nix agent node delete `ca.pem` by running:
+
+```bash
+rm -rf $(puppet config print localcacert)
```
Comment (m0dular) [inline README.md]: Could we remove the `-rf` options?  We shouldn't need recursive or force for a regular file.

**Review by m0dular — CHANGES_REQUESTED** [review]

**Review by m0dular — APPROVED** [review]

**Review by m0dular — CHANGES_REQUESTED** [review]
Looks great, but could we change the commit message so the changelog will be a little more readable?  The format should generally be something like

```
(SUP-3054) README updates

This commit does x, y, and z.
```
Where the first line has the JIRA key in parenthesis followed by a brief description, then a newline followed by a paragraph with more specific details about the commit.

**Comment by m0dular** [comment]
We should be able to change the commit message with `git commit --amend` followed by `git push -f`

**Review by m0dular — APPROVED** [review]

**Pull request merged**